met-web/src/components/publicDashboard/Dashboard.tsx

```diff
@@ -0,0 +1,94 @@
+import CompleteResponses from './KPI/CompleteResponses';
+import ResponsesWithComment from './KPI/ResponsesWithComment';
+import TotalResponses from './KPI/TotalResponses';
+import SubmissionTrend from './SubmissionTrend/SubmissionTrend';
+import { DashboardContext } from './DashboardContext';
+import SurveyBar from './SurveyBar.tsx';
+
+export const Dashboard = () => {
+    const navigate = useNavigate();
+    const { engagement, isEngagementLoading } = useContext(DashboardContext);
+
+    const handleReadComments = () => {
+        navigate(`/engagements/${engagement.id}/comments`);
+    };
+
+    return (
+        <Grid container direction="row" justifyContent="flex-start" alignItems="flex-start">
+            <Grid item xs={12}>
+                <ReportBanner engagement={engagement} isLoading={isEngagementLoading} />
+            </Grid>
+            <Grid
+                container
+                item
+                xs={12}
+                direction="row"
+                justifyContent={'flex-end'}
+                alignItems="flex-end"
+                m={{ lg: '1em 8em 2em 3em', sm: '2em', xs: '0.5em' }}
+            >
+                <Grid item xs={12} container justifyContent="flex-end">
+                    <MuiLink component={Link} to={`/engagements/${engagement.id}/view`}>
+                        {`<< Return to ${engagement.name} Engagement`}
+                    </MuiLink>
+                </Grid>
+
+                <Grid container item xs={12}>
+                    <MetPaper elevation={1} sx={{ padding: { md: '2em 2em 0 2em', sm: '1em', xs: '0.5em' } }}>
+                        <Grid
+                            container
+                            direction="row"
+                            justifyContent="flex-start"
+                            alignItems="flex-start"
+                            rowSpacing={3}
+                            sx={{ marginBottom: '2em' }}
+                        >
+                            <Grid item xs={12} sm={6}>
+                                <MetHeader1 textAlign={{ xs: 'center', sm: 'left' }}>What We Heard</MetHeader1>
+                            </Grid>
+                            <Grid
+                                item
+                                xs={12}
+                                sm={6}
+                                container
+                                direction={{ xs: 'column', sm: 'row' }}
+                                justifyContent="flex-end"
+                            >
+                                <PrimaryButton
+                                    data-testid="SurveyBlock/take-me-to-survey-button"
+                                    onClick={handleReadComments}
+                                >
+                                    Read Comments
+                                </PrimaryButton>
+                            </Grid>
+                            <Grid container spacing={3} item xs={12}>
```

Comment:
```
const components = [
  <TotalResponses />,
  <CompleteResponses />,
  <ResponsesWithComment />,
  <SubmissionTrend />,
  <SurveyBar />
];

const gridItems = components.map((component, index) => (
  <Grid key={index} item xs={12} sm={4}>
    {component}
  </Grid>
));

return (
  <Grid container spacing={3}>
    {gridItems}
  </Grid>
);

```
Do you wanna consider this style? incase if we add a new component , it will be easier..

Reply:
if its not worth it , feel free to ignore it..

Reply:
I see the value that this makes it shorter, and easily extendable by adding a component in the list. I would recommend keeping the current style because it offers more clarity and control. Components don't have to have the same xs and sm and our design might change for each.  Like submission Trend and Survey bar don't have sm={4}

Reply:
When dealing with Grid Items I would rather they are separate and easily independently controlled.